zipkin-cassandra/src/main/scala/com/twitter/cassie/Cluster.scala

```diff
@@ -59,12 +59,12 @@ class Cluster(seedHosts: Set[String], seedPort: Int, stats: StatsReceiver, trace
    * Returns a  [[com.twitter.cassie.KeyspaceBuilder]] instance.
    * @param name the keyspace's name
    */
-  def keyspace(name: String): KeyspaceBuilder = {
+  def keyspace(name: String, username: String, password: String): KeyspaceBuilder = {
```

Comment:
So if you try to authenticate with cassandra without the auth turned on, it just acts like a pass through.